src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -1893,22 +1893,7 @@ private static bool ReadConsoleOutputCJKSmall
                     }
                     else
                     {
-                        int charLength = LengthInBufferCells(contents[r, c].Character);
-                        if (charLength == 2)
```

Comment:
As I understand this code - somebody is screen scraping, there is a wide character (2 cells), and the font in use can't render the character, so conhost used 1 cell.
This seems like an extreme corner case, but the code seems correct to me and I don't any value in removing the code.
Also - the PR description should be improved - there is no link to an issue providing more detail.

Reply:
I'll update the PR description as I did not originally intend for this PR to be so extensive.  I simply started down the path of removing the -mta/-sta code since we weren't using it and then noticed the ConsoleFile code which wasn't being used, then the Font code... ugh...